workflow/scripts/get_clinical_guidelines.py

```diff
@@ -0,0 +1,239 @@
+                              right_on="HAPLOTYPE")
+        hap_df = hap_df.merge(self.activity_scores,
+                              how="left",
+                              left_on="Haplotype2",
+                              right_on="HAPLOTYPE",
+                              suffixes=("1", "2"))
+        hap_df["Genotype_activity"] = hap_df["ACTIVITY_SCORE1"] + hap_df[
+            "ACTIVITY_SCORE2"]
+
+        hap_df = hap_df.merge(self.clinical_guidelines,
+                              how="left",
+                              left_on=["gene", "Genotype_activity"],
+                              right_on=["Gene", "Activity"])
+
+        return hap_df
+
+    def get_wildtypes(self, hap_df):
+        hap_genes = list(hap_df.gene.values)
+        for gene in set(self.clinical_guidelines.Gene):
+            if hap_df.empty or gene not in hap_genes:
+                gene_df = pd.DataFrame({
+                    "gene": [gene],
+                    "Haplotype1": [gene + "-1"],
+                    "Haplotype2": [gene + "-1"],
+                    "HAPLOTYPE1": [gene + "-1"],
+                    "ACTIVITY_SCORE1": [1],
+                    "HAPLOTYPE2": [gene + "-1"],
+                    "ACTIVITY_SCORE2": [1],
+                    "Genotype_activity": [2.0]
+                })
+                gene_df = gene_df.merge(self.clinical_guidelines,
+                                        how="left",
+                                        left_on=["gene", "Genotype_activity"],
+                                        right_on=["Gene", "Activity"])
+                hap_df = pd.concat([hap_df, gene_df], ignore_index=True)
+        return hap_df
+
+
+def main():
+    haplotype_definitions = snakemake.params["haplotype_definitions"]
+    haplotype_activity = snakemake.params["haplotype_activity"]
+    hidden_haplotypes = snakemake.params["hidden_haplotypes"]
+    clinical_guidelines = snakemake.params["clinical_guidelines"]
+    variant_csv = snakemake.input["found_variants"]
+    output = snakemake.output["csv"]
+
+    ah = ArrangeHaplotype(variant_csv, haplotype_definitions,
+                          haplotype_activity, clinical_guidelines)
+
+    df = ah.get_clinical_guidelines_table()
+    df = ah.get_wildtypes(df)
+    columns = [
+        "gene", "Haplotype1", "Haplotype2", "HAPLOTYPE1", "ACTIVITY_SCORE1",
+        "HAPLOTYPE2", "ACTIVITY_SCORE2", "Genotype_activity", "Gene",
+        "Activity", "Guideline"
+    ]
+    if not df.empty:
+        hidden_haplotypes = pd.read_csv(hidden_haplotypes, sep="\t")
+        hidden_haplotypes["comb"] = hidden_haplotypes[[
+            "Haplotype1", "Haplotype2"
+        ]].apply(lambda x: "".join(sorted(x.tolist())), axis=1)
+        df["comb"] = df[["Haplotype1", "Haplotype2"
+                         ]].apply(lambda x: "".join(sorted(x.tolist())),
+                                  axis=1)
+        print(df["comb"])
+        print(hidden_haplotypes["comb"])
```

Comment:
Is this for debugging?

If it's not you should consider using logging https://docs.python.org/3/howto/logging.html